README.md

```diff
@@ -1,3 +1,5 @@
 # sfdcpoc
 
-## by shiv line added
\ No newline at end of file
+## by shiv line added
+
+## after protecting master branch in new branch
```

Comment:
Hey shiv, this line is unnecessary.

you could have used loops instead of writing this complicated recursive code.

Reply:
oh gotcha. Let me modify the code and revert.